### ISSUE TITLE
novnc: Drag left or right changes desktop and kills emulator UI

### ISSUE THREAD

**Issue by devilankur18** [budtmo/docker-android#107]
**Please make sure that it is an issue / a feature request. If it is a question / help wanted, please visit [our group chat](https://gitter.im/butomo1989/docker-android). Thank you!**

Operating System: Ubuntu 16.04

Docker Image: 6.0:latest

Docker Version: 17.12.1-ce

Docker-compose version (Only if you use it):  

Docker Command to start docker-android:  docker run --privileged -d -p 6080:6080 -p 4723:4723 -v /root/apks:/root/tmp -e DEVICE="Samsung Galaxy S6" -e APPIUM=true  -e APPIUM_HOST="127.0.0.1" -e APPIUM_PORT=4723 --name app butomo1989/docker-android-x86-6.0

  
## Expected Behavior

- The emulator is running on desktop 1. 
- Switch to desktop 2
- Come back to desktop 1
- Should see the emulator as working before

## Actual Behavior

The emulator is no more visible after switching back to desktop 1. Have to restart docker container.

**Comment by mehrdad-abdoli** [comment]
@devilankur18  right click and EXIT and confirm exit the emulator will be visible without need to restart

**Comment by devilankur18** [comment]
@mehrdad-abdoli that works !!

Can't we disable openbox or multiple desktop thingy all together. As it has no relevance for the usecase. And It happens with me often, I end up on another desktop and emulator is gone.

**Comment by 123dev** [comment]
@devilankur18 

Edit /etc/xdg/openbox/rc.xml
and delete / comment-out

```<mousebind button="Up" action="Click">
<action name="GoToDesktop">
  <to>previous</to>
</action>
</mousebind>
<mousebind button="Down" action="Click">
<action name="GoToDesktop">
  <to>next</to>
</action>
</mousebind>```

**Comment by budtmo** [comment]
Fixed in #168
